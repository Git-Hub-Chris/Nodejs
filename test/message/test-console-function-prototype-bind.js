'use strict';
require('../common');

Function.prototype.bind = function() {
  console.log('Should be logged to stdout.');
};

(() => {}).bind();
