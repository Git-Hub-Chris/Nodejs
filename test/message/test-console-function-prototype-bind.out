Should be logged to stdout.
